openapi/api-piattaforma-notifiche.md

```diff
@@ -0,0 +1,13 @@
+Procedura per importare le specifiche di PN:
```

Comment:
for curiosity: why are the specs being preprocessed using swagger-codegen-cli?

Reply:
Becouse the original API is splitted in many files one importing another. We used the swagger-codegen-cli to merge them in a single file. 
The original files use some openapi feature that our codegen do not implement yet (like the ones described in the api-piattaforma-notifiche.md file).
When we'll support all the missing feature, we could avoid this steps.